lib/manager/kustomize/__snapshots__/extract.spec.ts.snap

```diff
@@ -22,14 +20,16 @@ Array [
   Object {
     "currentValue": "v0.0.1",
     "datasource": "git-tags",
-    "depName": "https://moredhel/remote-kustomize.git",
```

Comment:
This one is a little confusing because `https://moredhel/` is a strange host to begin with. But if this is a `git-tags` resource, would it expect the protocol prefix and `.git` suffix or not?

Reply:
If you look at the [git-fetch manual](https://git-scm.com/docs/git-fetch#_git_urls), `.git` is always defined.
Isn't git-tags executing `git ls-remote` using the `lookupName` key?

Reply:
It's falling back to `depName` if `lookupName` is undefined. So `depName` is mandatory.

Reply:
ok, in this case we always define lookupName for dependency of type git-tags. So that should be fine to keep depName as it is?

Reply:
That would probably ok even if the test snapshots looking strange for now. We can fix that later.

Reply:
I agree, some of these tests are a bit confusing and I can definitely see some duplication of the logic.

I could probably simplify the whole thing and merge all these fixtures into 2 files (one empty, and one with all the variation of dependencies like https, ssh, etc.). What do you think? If you have manager in mind which can be use as model let me know.

Reply:
Sure, but the refactoring should be done in a seperate pr.

This pr needs at least one more test to get back  100% code coverage.